tests/osproc/tstdin.nim

```diff
@@ -11,9 +11,8 @@ doAssert fileExists(getCurrentDir() / "tests" / "osproc" / filename)
 var p = startProcess(filename, getCurrentDir() / "tests" / "osproc")
 p.inputStream.write("5\n")
 p.inputStream.flush()
-while true:
-  let line = p.outputStream.readLine()
-  if line != "":
-    echo line
-  else:
-    break
+
+var line = ""
+
+while p.outputStream.readLine(line.TaintedString):
```

Comment:
What? Well that conversion just defeats the purpose of `TaintedString`. `line` needs to be of type `TaintedString`. I think this shouldn't even be allowed in the language.